charts/datadog/templates/_container-agent.yaml

```diff
@@ -172,6 +172,16 @@
       readOnly: true
     {{- end }}
     {{- end }}
+    {{- if .Values.datadog.kubelet.hostCAPath }}
+    - name: kubelet-ca
+      {{- if eq .Values.targetSystem "linux" }}
+      mountPath: {{ include "datadog.kubelet.mountPath" . }}
+      {{- end }}
+      {{- if eq .Values.targetSystem "windows" }}
+      mountPath: {{ dir (include "datadog.kubelet.mountPath" .) }}
+      {{- end }}
+      readOnly: true
+    {{- end }}
```

Comment:
I think you can create an helper function for this mount, because it is duplicated in every containers

Reply:
Done